docs/_templates/layout.html

```diff
@@ -7,8 +7,9 @@
       <select onchange="location = this.value;"
               style="height: 30px; color: #27a54b; background-color: #403d3d;
                     border: 0px; box-shadow: none;">
-        <option value="https://scipp.github.io" selected>latest (unstable)</option>
-        <option value="https://scipp.github.io/release/0.6.0">v0.6</option>
+        <option value="" selected>Pick version</option>
```

Comment:
Great and simple solution for this! Can't believe we didn't think of this before...